### PR TITLE
RHTAPBUGS-1061 BASE_IMAGE_DIGESTS for multi platform

### PR DESCRIPTION
**Pull request by stuartwdouglas**
# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

**Review by arewm — APPROVED** [review]

File: task/buildah/0.1/buildah.yaml [hunk 1]
```diff
@@ -301,9 +304,6 @@ spec:
     image: $(params.BUILDER_IMAGE)
     computeResources: {}
     script: |
-      # Expose base image digests
-      buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}' | grep -v $IMAGE > $(results.BASE_IMAGES_DIGESTS.path)
-
```
Comment (tkdchen) [inline task/buildah/0.1/buildah.yaml:306]: Hi Stuart, why need this movement?

**Review by tkdchen — COMMENTED** [review]
LGTM

File: task/buildah/0.1/buildah.yaml (hunk 1 above)
Comment (stuartwdouglas) [reply task/buildah/0.1/buildah.yaml:306]: Because it needs to be run on the remote host, where the base images are actually pulled to.

**Review by tkdchen — APPROVED** [review]

**Pull request merged**